### ISSUE TITLE
Stealth Solar plant icon 

### ISSUE THREAD

**Issue by pyrog** [openinframap/styles#39]
The icon is briefly displayed at zoom level between [11](https://openinframap.org/#11/46.5605/5.7758) and [10.97](https://openinframap.org/#10.97/46.56/5.7767) when you play with the mouse wheel.

**Comment by russss** [comment]
Yeah, this is kind of a bug in Mapbox GL JS and how it interacts with my logic for determining how to display plant icons based on power output. There's no easy fix for it at the moment.
